scripts/migration/migrate_registration_extra.py

```diff
@@ -16,15 +17,38 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
+def migrate_file_representation(bad_file):
+    view_url = bad_file.get('viewUrl', '')
+    fid = view_url.split('/')[-2]
```

Comment:
An `IndexError` will occur here if `view_url` is `''`; is `viewUrl` guaranteed to be in `bad_file` and will it always have a `/`? If so, then you should use `bad_file['viewUrl']` rather than `.get`.